Makefile

```diff
@@ -8,44 +8,44 @@ help: ## Show this help
 	@grep -E '^[a-zA-Z\.\-]+:.*?## .*$$' $(MAKEFILE_LIST) | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
 
 .PHONY: paas
-paas:
+paas:  ## Sets the paas environment variables for over all PAAS deployment
 	$(eval PLATFORM=paas)
 	$(eval REGION=West Europe)
 	$(eval KEY_VAULT_SECRET_NAME=APPLY-QTS-APP-VARIABLES)
 
 .PHONY: aks
-aks:
+aks:  ## Sets environment variables for aks deployment
 	$(eval PLATFORM=aks)
 	$(eval REGION=UK South)
 	$(eval STORAGE_ACCOUNT_SUFFIX=sa)
 	$(eval KEY_VAULT_SECRET_NAME=APPLICATION)
 
 .PHONY: dev
-dev: paas
+dev: paas  ## Sets the paas environment variables for dev PAAS deployment
 	$(eval DEPLOY_ENV=dev)
-	$(eval AZURE_SUBSCRIPTION=s165-teachingqualificationsservice-development)
+	$(eval AZURE_SUBSCRIPTION=s189-teachingqualificationsservice-development)
```

Comment:
Why do we change the subscription for the PaaS environment?

Reply:
This has been corrected.